cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/task/AbstractPollTask.java

```diff
@@ -46,6 +46,6 @@ public boolean cancelled() {
             return false;
         }
         PollGroup pollGroup = InMemoryStateStore.getStack(authenticatedContext.getCloudContext().getId());
-        return pollGroup != null && CANCELLED.equals(pollGroup);
+        return CANCELLED.equals(pollGroup);
```

Comment:
is this cannot be null?

Reply:
Can be but unnecessary, equals also checks it